### DISCUSSION TITLE
RAF LVL 3 Changes

### DISCUSSION THREAD

**Discussion by TelumehtarCC** [chromiecraft/chromiecraft#6899] [Suggestions]
Hey, not to sound ungrateful but by the time you have 3 friends who hit the RAF bonus of 70. You're... really not in need of a Runecloth bag. Can it be a Primal Mooncloth bag or another cosmetic/pet?

**Comment by 55Honey** [comment]
Hi @TelumehtarCC ,
thanks for your suggestion!

After the next restart, elixirs from RAF rewards will be replaced with those from the 50-60 level range. There's a gonna be 16-slot bag going forward, too.
